### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by codergirl01**

### CODE REVIEW

**Comment by Briantam0422** [comment]
Fix merge conflicts with Wendy and force update the branch. All good to go now.

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   },
   "dependencies": {
     "@auth/prisma-adapter": "^1.4.0",
-    "@prisma/client": "^5.10.2",
+    "@prisma/client": "^4.4.0",
```
Comment (ialej001) [inline package.json:17]: Why is this being downgraded?

File: package.json [hunk 2]
```diff
@@ -52,9 +53,10 @@
     "postcss": "^8.4.34",
     "prettier": "^3.2.5",
     "prettier-plugin-tailwindcss": "^0.5.11",
-    "prisma": "^5.10.2",
+    "prisma": "^4.4.0",
```
Comment (ialej001) [inline package.json:56]: Same with this, why the downgrade?

File: package.json [hunk 3]
```diff
@@ -52,9 +53,10 @@
...
     "tailwindcss": "^3.4.1",
-    "typescript": "^5.3.3"
+    "typescript": "^5.3.3",
+    "node": "^18.20.0"
```
Comment (ialej001) [inline package.json:59]: I think this doesn't belong here. Node is typically a globally installed package, not project level.

**Review by ialej001 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (codergirl01) [reply package.json]: Updated now

File: package.json (hunk 2 above)
Comment (codergirl01) [reply package.json]: Updated

File: package.json (hunk 3 above)
Comment (codergirl01) [reply package.json]: Removed this

**Review by ialej001 — APPROVED** [review]

File: src/pages/profile/index.tsx
Comment (Briantam0422) [inline src/pages/profile/index.tsx:1]: Sorry for forgetting to mention it. As we have the user profile page now, we can delete this profile page.

**Review by Briantam0422 — CHANGES_REQUESTED** [review]

**Review by Briantam0422 — APPROVED** [review]

**Pull request merged**